src/bp/ui-studio/src/web/actions/index.ts

```diff
@@ -428,6 +428,14 @@ export const refreshIntents = () => dispatch => {
   })
 }
 
+export const entitiesReceived = createAction('ENTITIES/RECEIVED')
+export const refreshEntities = () => dispatch => {
+  // tslint:disable-next-line: no-floating-promises
```

Comment:
is this floating ? you have a .then

Reply:
but no catch. I kept the same structure as all others actions, but a refactor would be welcome